VDA-StorageNode-Registry/contracts/StorageNodeRegistry.sol

```diff
@@ -0,0 +1,1098 @@
+    ) external override {
+        uint nodeId = _didNodeId[didAddress];
+
+        // Check whether didAddress was registered before
+        if (nodeId == 0 || _nodeUnregisterTime[nodeId] != 0) {
+            revert InvalidDIDAddress();
+        }
+
+        if (unregisterDateTime < (block.timestamp + 28 days)) {
+            revert InvalidUnregisterTime();
+        }
+
+        bytes memory params = abi.encodePacked(didAddress, unregisterDateTime);
+        verifyRequest(didAddress, params, requestSignature, requestProof);
+
+        _nodeUnregisterTime[nodeId] = unregisterDateTime;
+
+        emit RemoveNodeStart(didAddress, unregisterDateTime);
+    }
+
+    /**
+     * @notice Release staked token to the requestor of `removeNodeComplete()` function
+     * @dev Internal function. Called inside of `removeNodeComplete()` function
+     * @param didAddress DID address that calls the `removeNodeComplete()` function
+     * @param to EOA wallet that will received the released token
+     */
+    function releaseToken(address didAddress, address to) internal {
+        uint totalAmount = _stakedTokenAmount[didAddress];
+
+        if (totalAmount != 0) {
+            IERC20Upgradeable(vdaTokenAddress).transfer(to, totalAmount);
+            _stakedTokenAmount[didAddress] = 0;
+        }        
+    }
+
+    /**
+     * @dev see { IStorageNodeRegistry }
+     */
+    function removeNodeComplete(
+        address didAddress,
+        bytes calldata requestSignature,
+        bytes calldata requestProof
+    ) external override {
+        uint nodeId = _didNodeId[didAddress];
+        {
+            if (nodeId == 0) {
+                revert InvalidDIDAddress();
+            }
+
+            if (_nodeUnregisterTime[nodeId] == 0 || _nodeUnregisterTime[nodeId] > block.timestamp) {
+                revert InvalidUnregisterTime();
+            }
+
+            bytes memory params = abi.encodePacked(didAddress);
+            verifyRequest(didAddress, params, requestSignature, requestProof);
+        }
+
+        // Release staked token
+        releaseToken(didAddress, tx.origin);
+
+        // Clear registered information
+        StorageNode storage nodeInfo = _nodeMap[nodeId];
+
+        --_datacenterInfo[nodeInfo.datacenterId].nodeCount;
+
+        _countryNodeIds[nodeInfo.countryCode].remove(nodeId);
+        _regionNodeIds[nodeInfo.regionCode].remove(nodeId);
+        delete _endpointNodeId[nodeInfo.endpointUri];
+        delete _didNodeId[didAddress];
+        delete _nodeMap[nodeId];
+
+        delete _nodeUnregisterTime[nodeId];
+
+        emit RemoveNodeComplete(didAddress);
+    }
+
+    /**
+     * @notice Create tuple for StorageNode with status
+     * @param nodeId StorageNode ID created by `addNode()` function
+     * @return StorageNode StoargeNode struct
+     * @return string Status string
+     */
+    function getNodeWithStatus(uint nodeId) private view returns(StorageNode memory, string memory) {
```

Comment:
Please rename to `getNodeById()`

Reply:
This is private function and not used externally.
And this functions returns node and it's status. So named it so. If we rename this function by `getNodebyId()`, it seems like it returs a node that is matched with the inputed nodeId.
Let me know your thougts.

Reply:
Ok, thats makes senses.

Leave as is.